chapters/testing.Rmd

```diff
@@ -731,21 +752,20 @@ $ coverage report -m
 ```
 
 ```text
-Name            Stmts   Miss  Cover   Missing
----------------------------------------------
-countwords.py      20      8    60%   19-21, 25-31
-plotcounts.py      46     27    41%   41-47, 67-69, 74-90, 94-104
-test_zipfs.py      32      0   100%
-utilities.py        7      4    43%   24-27
----------------------------------------------
-TOTAL             105     39    63%
+bin/countwords.py      20      7    65%   25-26, 30-37
+bin/plotcounts.py      58     37    36%   48-55, 75-77, 81-82, 
+                                          86-118, 121-139
+bin/test_zipfs.py      31      0   100%
+bin/utilities.py        8      5    38%   18-22
+-------------------------------------------------
+TOTAL                 117     49    58%
```

Comment:
Thanks for updating the output from `$ coverage report -m`. Would you also be able to update Figure 11.1? It's a screenshot of the `$ coverage html` output for `countwords.py` and it currently shows a very old version of that script.